examples/trivial/BUILD

```diff
@@ -27,3 +27,16 @@ ktlint_config(
     experimental_rules_enabled = False,
     visibility = ["//visibility:public"],
 )
+
+sh_test(
```

Comment:
There's a skylib macro for this that asserts if something builds.

Probably worth using just to minimize the surface area of rules Kotlin a bit.

Reply:
Needs to run to test it. Building is easy -- but the kotlin/jvm integration will only fail if it doesn't run.